### PR TITLE
ECS Services: Explorer, Transfer and ConfigServer

### PR DESCRIPTION
**Pull request by mschirbel**
## Related Issue Or Context

Closes: #<issue>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [x] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: ecs/task_definition-config-server-STAGE.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      "portMappings": [
+          {
+              "hostPort": 8126,
+              "protocol": "tcp",
+              "containerPort": 8126
+          }
+      ],
+      "command": [],
+      "cpu": 0,
+      "environment": [
```
Comment (MakMuftic) [inline ecs/task_definition-config-server-STAGE.json:67]: I think this is missing:
`HOST`
`PORT`
`SSM_PARAMETER_NAME`

For ssm parameter I created new one called: `/chainbridge/chainbridge-ui/configuration` that should be used

Comment (MakMuftic) [reply ecs/task_definition-config-server-STAGE.json:67]: @mschirbel @enemycnt

Comment (mschirbel) [reply ecs/task_definition-config-server-STAGE.json:75]: for the config-server we have:

```
"environment": [
          {
            "value": "config-server",
            "name": "DD_SERVICE"
          },
          {
            "name": "PORT",
            "value": "8000"
          },
          {
            "name": "HOST",
            "value": "localhost"
          }
        ],
        "secrets": [ 
           { 
              "name": "SSM_PARAMETER_NAME",
              "valueFrom": "arn:aws:ssm:us-east-2:852551629426:parameter/chainbridge/chainbridge-ui-configuration"
           }
        ]
```

Comment (MakMuftic) [reply ecs/task_definition-config-server-STAGE.json:75]: looks good :+1:

File: .github/workflows/deploy_ecs-transfer-ui_STAGE.yaml [hunk 2]
```diff
@@ -46,13 +46,15 @@ jobs:
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
         run: |
+          cp yarn.lock packages/transfer-widget
```
Comment (MakMuftic) [inline .github/workflows/deploy_ecs-transfer-ui_STAGE.yaml:49]: @mschirbel I think here is the problem, we need to use root Dockerfile - so we can remove these two lines

**Comment by MakMuftic** [comment]
I changed the target branch to `main` for now. We will change for tagged versions to be deployed as soon as we have a stable `dev` environment.

**Review by MakMuftic — APPROVED** [review]

**Review by enemycnt — APPROVED** [review]

**Pull request merged**